src/aws.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * AWS bits and bobs.
+ *
+ * @module aws
+ */
+import { NodeHttpHandler } from "@smithy/node-http-handler";
```

Comment:
Suggestion: Add `@smithy/node-http-handler` to the list of direct dependencies in package.json. Currently it's an indirect dependency through `@aws-sdk/client-s3`.

Reply:
Whoops. I meant to but forgot.

Reply:
Done in reworked commit 911dbf5b.